### DISCUSSION TITLE
relationships

### DISCUSSION THREAD

**Discussion by dustymc** [ArctosDB/arctos#8386] [Ideas]
From https://github.com/ArctosDB/arctos/issues/8384 - do we need some ability to do something that we can't do now?

Suggest starting with functional needs, not models or possible implementations.

Current CT: https://arctos.database.museum/info/ctDocumentation.cfm?table=ctid_references

**Comment by Jegelewicz** [comment]
[GloBI](https://www.globalbioticinteractions.org/) is a public repository that aggregates biotic relationships. GloBI relies on the [OBO Relation Ontology](https://obofoundry.org/ontology/ro.html). Why not follow what GloBI does for biological interactions?

There are plenty of relationships where the related object/specimen is not cataloged, so we have also recorded those in a multitude of attributes with no taxonomic control. I think that should all be considered as part of this as well, because finding "relationships" in Arctos requires searching in many places (identifications, attributes, identifiers).

**Comment by dustymc** [comment]
>Why not follow what GloBI does 

It's a very different model - which doesn't mean we can't do nearly anything, as long as we keep our reality in sight.

>[OBO Relation Ontology](https://obofoundry.org/ontology/ro.html).

There's one obstacle: Adding some meteorite-or-whatever relationship would, under a 'just use the ontology' model, first require adding that to some ontology and/or adding that ontology to the mashup. Arctos scales broader than almost anything else. (This still absolutely doesn't mean we can't do something awesome, just that being awesome-r would be a bit more work at a bit different scale than we're used to.)

>related object/specimen is not cataloged

That's the core difference, Arctos is not so simple as the semantic triple that everyone in ontology-land expects, and that may or may not be an obstacle, and it may or may not be surmountable if it is.

https://github.com/ArctosDB/BackBurner/issues/25 - there's clearly an ontology community, there's clearly interest in joining it, someone write the proposal if that's where The Community wants to go!
